tests/new_unit_tests/Commands/Auth/CloneCommandTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Pantheon\Terminus\UnitTests;
+
+use Pantheon\Terminus\UnitTests\Commands\CommandTestCase;
+use Pantheon\Terminus\UnitTests\Commands\Env\MockCloneCommand;
+use Terminus\Models\Site;
+
+class CloneCommandTest extends CommandTestCase
+{
+    protected $command;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->command = new MockCloneCommand();
+        $this->site = $this->getMockBuilder(Site::class)
```

Comment:
This is redundant. `CommandTestCase::setUp()` does this exact operation.
https://github.com/pantheon-systems/terminus/blob/master/tests/new_unit_tests/Commands/CommandTestCase.php#L258-L260